### PR TITLE
What is AWS, why is AWS popular?

### LINKED ISSUE #2386: What is AWS, why is AWS popular?
We ask that you please be patient as our team works through approving and publishing all articles/tutorials in a timely manner. Allow 1-3 days for a topic to be reviewed and/or approved - allow 3-7 days for an article to be reviewed and/or published.

Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles to go over.

**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**NOTE: We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

*We're looking for the first 2-3 paragraphs of the article that appropriately summarize what your article will be about.*

### Proposed title of the article
What is AWS and reasons why AWS is so popular

### Introduction paragraph (2-3 paragraphs):
AWS stands for Amazon Web Services, a subsidiary of Amazon, the tech giant. AWS provided cloud computing and APIs services on-demand for organizations to run and host online infrastructures and applications. It offers a wide variety of computing powers, databases, storage, content delivery platforms, networking, virtualization, analytics, security, and many services that easily scalable and available across businesses and organizations of all levels.

### A brief history of AWS and how it came to be

Aws was first launched in 2002 internally by amazon to provide tools to developers to incorporate the features of amzon.com to the website.

In 2004 The first AWS service was launched to the public. The service was called simple queuing service (SQS). It provided just simple queue services.

>Some omitted paragraphs

In 2013, AWS saw one of its major breakthroughs, the Central Intelligence Agency (CIA), moved its infrastructure to AWS. This was a huge success. In 2015, AWS released its profit figure to the public, with a record of 4.6 billion for the previous year (2014). with an expected growth of about 49% that is 6.2 billion in 2015. And in 2016, AWS surpassed its 10 billion revenue target.

Today, AWS has a ton of services and certifications. There is a massive demand for AWS services these days. The AWS cloud service platform is now estimated to be used by more than 45% of the global market. An average of 1 of the 3 website you visits uses AWS services.

### A recap of cloud service

Think of a personal use case of a cloud service. Just like iCloud and Dropbox, AWS is a cloud service provider. In your past, you may have used cloud computing to store pictures, videos, and documents. However, that is only a very small subset of what cloud computing is.

>Some omitted paragraphs

In this scenario, you have the following main cloud advantage.

High availability and fault tolerance

### How AWS Works as a cloud service provider

Think of an enterprise company that is not using cloud services such as AW. this means the company has on-premise data centers and servers for handling and storing their data. When a customer wants to access the company's software or a service, they're accessing resources on the company's servers.

Let's say in 2019, about 3000 users accessed this software, so the company has three severs to power up its software to these users. When the number of user base grows in the next year, let's say to 6000. The three servers the company has will not be enough to handle that significant number of additional users. The company will be required to add a cost and space to double the number of servers on its on-premise data center to handle the load of having more users. In this case, they have to research what type of servers they need, make an order, wait for delivery, set them up, install operating systems software, test them and get them all up and running.

>Some omitted paragraphs

This is the biggest problem that cloud computing provides, such as AWS, solves. The company no longer has to project or estimate future growth naturally to set a number of servers. As the user base grows, AWS can automatically and instantly add additional servers and resources according to the number of users accessing the company's services. AWS, within no time it automatically upgrades functionality and features for you.

>Some omitted paragraphs

### Key takeaways: 
A brief history of AWS and how it came to be
A recap of cloud service
How AWS Works as a cloud service provider
How popular is AWS
Reasons why AWS is popular

### References: 
Please list links to any published content/research that you intend to use to support/guide this article.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by mosesreigns**
### Article checklist

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**
      
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
Fixes #2386

### CODE REVIEW

File: content/articles/what-is-aws-and-why-is-aws-so-popular/index.md.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+AWS was categorized as a platform with operating systems such as Linux and Windows and container technologies such as Docker and Kubernetes. However, about 26% of the respondents/ developers Used AWS for the development of their work.
+
+![Popular platforms](popular-platforms.png)
+
+[Image source](https://insights.stackoverflow.com/survey/2020#technology-platforms-all-respondents5)
+
+The above findings were much higher than AWS's close competitor Google Cloud Platform and Microsoft Azure.
+
+![Cloud service providers](cloud-service-providers.png)
```
Comment (WanjaMIKE) [inline content/articles/what-is-aws-and-why-is-aws-so-popular/index.md.md:75]: Kindly, add the correct image links. Thanks

**Comment by adrianmurage** [comment]
@WanjaMIKE , I see you have plenty number of PRs assigned. I would recommend you wait till all your PRs are published before taking on new PRs. I will be reviewing this PR. Thank you :)

cc: @hectorkambow

**Comment by WanjaMIKE** [comment]
@adrianmurage No, Can't do. I am already in progress. Kindly, wait for a new PR. Thanks

**Comment by mosesreigns** [comment]
@WanjaMIKE  I've added the Image links

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by hectorkambow** [comment]
@mosesreigns  NIce work 👍 
Please go over the edits I made during my final to improve future articles 🚀 

cc @WanjaMIKE 
Do not forget your peer reviewer line. 

Please go through the formatting changes I made as well 👍 - thank you

**Comment by WanjaMIKE** [comment]
Noted @hectorkambow

**Pull request merged**